qi-lib/flow/core/compiler/1000-qi0.rkt

```diff
@@ -28,6 +28,21 @@
     ;;;; Core language forms ;;;;
     ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+    ;; A note regarding symbolic aliases like ~>, ⏚ and △:
```

Comment:
RE: dropping symbolic aliases in 3d661fb (add back symbolic aliases that got dropped at some point, 2025-01-15), I think they disappeared in c6096258 (extract more non-core forms as macros, 2022-08-25), which is contained in v4.0!

Reply:
At least, `git log -S <alias> upstream/deforest-all-the-things` finds that commit for both AND and OR; for NOT, I haven't managed to find the removal yet.

Reply:
Whoops, guess they've been gone quite a while 😬 Actually, they might have slipped out as early as the addition of Syntax Spec (was that 3.0?), as it's quite possible we just neglected to add the aliases to the `provide` form in the expander at the time. Prior to that, just having the datum matching of the symbolic variants in the original `flow` macro was sufficient for them to be part of the "core"/flat language. While the rules of that original macro carried over to `qi0->racket`, the ground truth for the core language switched to being the syntax spec in the expander, and it just might have slipped through the cracks then.

Reply:
That explains why the "pair with MB" commit came up a few times in my search. I definitely wasted too much time looking though: I probably should have tried `git bisect`.